PlanningPoker.Shared/ItemEstimateDTO.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PlanningPoker.Shared
+{
+    class ItemEstimateDTO
+    {
+        public int Id { get; set; }
+
+        public string ItemTitle { get; set; }
```

Comment:
We don't save Description here?

Reply:
According to our google sheet with all the entities etc,  an `ItemEstimate` only contains title and estimate.